xcube_cci/cciodp.py

```diff
@@ -1092,6 +1092,12 @@ async def _extract_metadata_from_odd_url(self, session: aiohttp.ClientSession,
         xml_text = await resp.read()
         return _extract_metadata_from_odd(etree.XML(xml_text))
 
+    def _determine_fill_value(self, dtype):
+        if np.issubdtype(dtype, np.integer):
+            return np.iinfo(dtype).max
```

Comment:
I don't get this (seems to have no test as well): For example `np.iinfo(np.uint8).max == 255`, this means 255 will become  NaN after decoding, however 255 was a valid Value before.